isis/tests/AutoseedTests.cpp

```diff
@@ -0,0 +1,90 @@
+#include <iostream>
+#include <QTemporaryFile>
+#include <QTemporaryDir>
+
+#include "autoseed.h"
+#include "findimageoverlaps.h"
+#include "footprintinit.h"
+
+#include "Cube.h"
+#include "CubeAttribute.h"
+#include "ImageOverlapSet.h"
+#include "PixelType.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "PvlKeyword.h"
+#include "TestUtilities.h"
+#include "FileName.h"
+
+#include "Fixtures.h"
+
+#include "gmock/gmock.h"
+
+using namespace Isis;
+
+static QString FOOTPRINT_XML = FileName("$ISISROOT/bin/xml/footprintinit.xml").expanded();
+static QString OVERLAP_XML = FileName("$ISISROOT/bin/xml/findimageoverlaps.xml").expanded();
+static QString AUTOSEED_XML = FileName("$ISISROOT/bin/xml/autoseed.xml").expanded();
```

Comment:
Just to make it explicit, it seems the best thing to do here is to avoid apps depending on other apps and just bake these outputs into the fixture. I hate saying that since there was a lot of work going in the opposite direction.

Reply:
I mean, won't the fixtures use the app calls? So no big deal and it needed to be done anyway, right?

Reply:
I'd hard code the output into the fixtures instead of calling the apps. The footprint BLOB and overlap file format are stable "standards".

Reply:
:+1: I'm not sure the best way to go about doing that but I agree

Reply:
This is going to be handled in a separate PR to avoid having this PR grow further. The test works currently, but has some dependencies.